src/components/bookForm.js

```diff
@@ -20,32 +20,28 @@ const BookForm = () => {
   };
 
   return (
-    <div>
-      <h2>Add a New Book</h2>
-      <form>
+    <section className="form-container">
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form className="add-form">
         <input
+          className="input title-input"
           type="text"
           placeholder="Title"
           value={newBook.title}
           onChange={(e) => setNewBook({ ...newBook, title: e.target.value })}
         />
         <input
+          className="input title-input"
           type="text"
           placeholder="Author"
           value={newBook.author}
           onChange={(e) => setNewBook({ ...newBook, author: e.target.value })}
         />
```

Comment:
[OPTIONAL] Goodjob working on the project requirements  therefore I not if the previous code reviewer noticed this but the add and remove book functionality is not working it would be nice if you had a look into this because it is an importart project requirement I marked this as optional because the previous code reviewer did not mention this to you so kindly correct this issue.